packages/fxa-auth-server/lib/db/index.js

```diff
@@ -162,31 +164,10 @@ module.exports = (config, log, Token, UnblockCode = null) => {
   DB.prototype.sessions = async function (uid) {
     log.trace('DB.sessions', { uid });
     const getMysqlSessionTokens = async () => {
-      let sessionTokens = await RawSessionToken.findByUid(uid);
-      if (!MAX_AGE_SESSION_TOKEN_WITHOUT_DEVICE) {
-        return sessionTokens;
-      }
-
-      const expiredSessionTokens = [];
-
-      // Filter out any expired sessions
-      sessionTokens = sessionTokens.filter((sessionToken) => {
-        if (sessionToken.deviceId) {
-          return true;
-        }
-
-        if (
-          sessionToken.createdAt >
-          Date.now() - MAX_AGE_SESSION_TOKEN_WITHOUT_DEVICE
-        ) {
-          return true;
-        }
-
-        expiredSessionTokens.push(
-          Object.assign({}, sessionToken, { id: sessionToken.tokenId })
-        );
-        return false;
-      });
+      const { sessionTokens, expiredSessionTokens } = filterExpiredTokens(
```

Comment:
Moved to fxa-shared. This chunk of code was overlooked on the previous refactor for the server side support of connected services.